### PR TITLE
Feat/prashanth fuel export 823 824

### PR DESCRIPTION
**Pull request by prv-proton**
#823 
#824 
#825

### CODE REVIEW

File: frontend/src/views/FuelExports/AddEditFuelExports.jsx [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+    async (params) => {
+      if (params.column.colId === 'fuelType') {
+        const options = optionsData?.fuelTypes
+          ?.find((obj) => params.node.data.fuelType === obj.fuelType)
+          ?.fuelCategories.map((item) => item.fuelCategory)
+        if (options.length === 1) {
+          params.node.setDataValue('fuelCategory', options[0])
+        }
+      }
+      // if (
```
Comment (AlexZorkin) [inline frontend/src/views/FuelExports/AddEditFuelExports.jsx:120]: Is this commented code required?

File: backend/lcfs/web/api/fuel_export/services.py [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+        return FuelExportSchema.model_validate(created_equipment)
+
+    @service_handler
+    async def delete_fuel_export(self, fuel_export_id: int) -> str:
+        """Delete a fuel supply record"""
+        return await self.repo.delete_fuel_export(fuel_export_id)
+
+    # TODO Left here for example for version tracking work
+    @service_handler
+    async def create_supplemental_fuel_export(
```
Comment (AlexZorkin) [inline backend/lcfs/web/api/fuel_export/services.py:441]: Thank you for including this supplemental code for future use.

**Review by AlexZorkin — COMMENTED** [review]
Looking great. Let me know when it's out of draft plz. We will need tests for the new backend, and frontend code. ty

**Comment by prv-proton** [comment]
Yes @AlexZorkin, adding tests and fixing issues found after code merge. Thanks for the review

File: frontend/src/views/FuelExports/AddEditFuelExports.jsx (hunk 1 above)
Comment (prv-proton) [reply frontend/src/views/FuelExports/AddEditFuelExports.jsx]: moved the logic to backend.

**Review by AlexZorkin — APPROVED** [review]
Looks great. Last schedule for cr complete!

**Pull request merged**